components/snackbar/styles.css

```diff
@@ -0,0 +1,31 @@
+:root{
```

Comment:
make this a seperate file, it will be easy to edit because these colours will be used in every component.

Reply:
Thanks @saifudinbadani . I have created it